shared/src/main/java/org/pac4j/lagom/Authorizers.java

```diff
@@ -0,0 +1,95 @@
+package org.pac4j.lagom;
+
+import org.pac4j.core.authorization.authorizer.Authorizer;
+import org.pac4j.core.authorization.authorizer.IsAnonymousAuthorizer;
+import org.pac4j.core.authorization.authorizer.IsAuthenticatedAuthorizer;
+import org.pac4j.core.authorization.authorizer.RequireAnyRoleAuthorizer;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.profile.CommonProfile;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Fluent API for creating authorizers in implementation Lagom services.
+ *
+ * @author Sergey Morgunov
+ * @since 1.0.0
+ */
+public final class Authorizers {
+
+    public static <U extends CommonProfile> Authorizer<U> authenticated() {
+        return new IsAuthenticatedAuthorizer<>();
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anonymous() {
+        return new IsAnonymousAuthorizer<>();
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anyRole(String ... roles) {
+        return new RequireAnyRoleAuthorizer<>(roles);
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anyRole(List<String> roles) {
+        return new RequireAnyRoleAuthorizer<>(roles);
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anyRole(Set<String> roles) {
+        return new RequireAnyRoleAuthorizer<>(roles);
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> and(Authorizer<U>... authorizers) {
+        return new AndAuthorizer<>(Arrays.asList(authorizers));
+    }
+
+    public static <U extends CommonProfile> OrAuthorizer<U> or(Authorizer<U>... authorizers) {
+        return new OrAuthorizer<>(Arrays.asList(authorizers));
+    }
+
+    /**
+     * The conjunction of authorizers.
+     * @param <U> Type of profile
+     */
+    private static class AndAuthorizer<U extends CommonProfile> implements Authorizer<U> {
+
+        private final List<Authorizer<U>> authorizers;
+
+        AndAuthorizer(List<Authorizer<U>> authorizers) {
+            this.authorizers = authorizers;
+        }
+
+        @Override
+        public boolean isAuthorized(WebContext context, List<U> profiles) {
+            for (Authorizer<U> authorizer : authorizers) {
+                if (!authorizer.isAuthorized(context, profiles)) return false;
+            }
+            return true;
+        }
+
+    }
+
+    /**
+     * The disjunction of authorizers.
+     * @param <U> Type of profile
+     */
+    private static class OrAuthorizer<U extends CommonProfile> implements Authorizer<U> {
```

Comment:
I think this should into the `pac4j-core` project.

Reply:
I created PR https://github.com/pac4j/pac4j/pull/1199
1. Move code to `pac4j-core`.
2. Mark this class how `deprecated`. (until release 3.4.0)

Reply:
👌 I will remove this class and migrate tests after merging https://github.com/pac4j/pac4j/pull/1199

Reply:
Done